detox/scripts/postinstall.js

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env node
+
+// Javascript for windows support.
```

Comment:
Unnecessary comment

Reply:
More clearly `Using JS instead of shell script to avoid breaking on windows`, but I'm happy to remove it!